packages/cli/__tests__/zostso/__system__/stop/cli.zos-tso.stop.address-space.system.test.ts

```diff
@@ -32,16 +32,18 @@ describe("zos-tso start address-space", () => {
 
     it("should throw an error if servlet key parameter is not provided", async () => {
         const response = runCliScript(__dirname + "/__scripts__/address-space/as_error_stop.sh", TEST_ENVIRONMENT);
-        expect(response.status).toBe(1);
         expect(response.stdout.toString()).toBe("");
-        expect(response.stderr.toString()).toMatchSnapshot();
+        expect(response.stderr.toString()).toContain('IZUG1126E: z/OSMF cannot correlate the request for ' +
```

Comment:
Should we flip the order to be 1st stderr and 2nd stdout (same question as above)?

Reply:
The previous comment also applies in this case 😋